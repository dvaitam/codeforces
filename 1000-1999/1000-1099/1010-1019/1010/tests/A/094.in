3
16
5 10 6
8 10 4
