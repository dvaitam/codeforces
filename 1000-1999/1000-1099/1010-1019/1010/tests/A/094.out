29.1499118166
